diff --git a/board/safety/safety_gm.h b/board/safety/safety_gm.h
index 6d7c668..995a87e 100644
--- a/board/safety/safety_gm.h
+++ b/board/safety/safety_gm.h
@@ -19,8 +19,8 @@ const int GM_MAX_GAS = 3072;
 const int GM_MAX_REGEN = 1404;
 const int GM_MAX_BRAKE = 350;
 const CanMsg GM_TX_MSGS[] = {{384, 0, 4}, {1033, 0, 7}, {1034, 0, 7}, {715, 0, 8}, {880, 0, 6},  // pt bus
-                             {161, 1, 7}, {774, 1, 8}, {776, 1, 7}, {784, 1, 2},   // obs bus
-                             {789, 2, 5},  // ch bus
+                             {161, 1, 7}, {774, 1, 8}, {776, 1, 7}, {784, 1, 2}, {384, 1, 4}, {715, 1, 8}, {880, 1, 6}, {789, 1, 5}, {0x104c006c, 1, 3},  // obs bus
+                             {789, 2, 5},   // ch bus
                              {0x104c006c, 3, 3}, {0x10400060, 3, 5}};  // gmlan
 
 // TODO: do checksum and counter checks. Add correct timestep, 0.1s for now.
@@ -34,10 +34,21 @@ AddrCheckStruct gm_addr_checks[] = {
 #define GM_RX_CHECK_LEN (sizeof(gm_addr_checks) / sizeof(gm_addr_checks[0]))
 addr_checks gm_rx_checks = {gm_addr_checks, GM_RX_CHECK_LEN};
 
+bool pt_ecu_interceptor = false;
+
 static int gm_rx_hook(CAN_FIFOMailBox_TypeDef *to_push) {
 
   bool valid = addr_safety_check(to_push, &gm_rx_checks, NULL, NULL, NULL);
 
+  if (valid && (GET_BUS(to_push) == 1)) {
+  int addr = GET_ADDR(to_push);
+  // Check for ASCM ECU Interceptor Status
+  // TODO: Add second check for chas bus interceptor. If only PT bus has an interceptor, we should only allow steering commands
+  if (addr == 885) {
+      pt_ecu_interceptor = true;
+    }
+  }
+
   if (valid && (GET_BUS(to_push) == 0)) {
     int addr = GET_ADDR(to_push);
 
@@ -89,11 +100,11 @@ static int gm_rx_hook(CAN_FIFOMailBox_TypeDef *to_push) {
       }
     }
 
-    // Check if ASCM or LKA camera are online
+   // Check if ASCM or LKA camera are online
     // on powertrain bus.
     // 384 = ASCMLKASteeringCmd
     // 715 = ASCMGasRegenCmd
-    generic_rx_checks(((addr == 384) || (addr == 715)));
+    generic_rx_checks(((addr == 384) || (addr == 715)) && !pt_ecu_interceptor);
   }
   return valid;
 }
@@ -127,7 +138,7 @@ static int gm_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
   bool current_controls_allowed = controls_allowed && !pedal_pressed;
 
   // BRAKE: safety check
-  if (addr == 789) {
+  if ((addr == 789) || (addr == 788)) {
     int brake = ((GET_BYTE(to_send, 0) & 0xFU) << 8) + GET_BYTE(to_send, 1);
     brake = (0x1000 - brake) & 0xFFF;
     if (!current_controls_allowed) {
@@ -141,7 +152,7 @@ static int gm_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
   }
 
   // LKA STEER: safety check
-  if (addr == 384) {
+  if ((addr == 384) || (addr == 383)) {
     int desired_torque = ((GET_BYTE(to_send, 0) & 0x7U) << 8) + GET_BYTE(to_send, 1);
     uint32_t ts = microsecond_timer_get();
     bool violation = 0;
@@ -189,7 +200,7 @@ static int gm_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
   }
 
   // GAS/REGEN: safety check
-  if (addr == 715) {
+  if ((addr == 715) || (addr == 714)) {
     int gas_regen = ((GET_BYTE(to_send, 2) & 0x7FU) << 5) + ((GET_BYTE(to_send, 3) & 0xF8U) >> 3);
     // Disabled message is !engaged with gas
     // value that corresponds to max regen.
