diff --git a/selfdrive/car/car_helpers.py b/selfdrive/car/car_helpers.py
index 980b207f..5a66daae 100644
--- a/selfdrive/car/car_helpers.py
+++ b/selfdrive/car/car_helpers.py
@@ -14,7 +14,8 @@ EventName = car.CarEvent.EventName
 
 
 def get_startup_event(car_recognized, controller_available, fw_seen):
-  if comma_remote and tested_branch:
+  #if comma_remote and tested_branch:
+  if True:
     event = EventName.startup
   else:
     event = EventName.startupMaster
@@ -169,6 +170,7 @@ def fingerprint(logcan, sendcan):
 def get_car(logcan, sendcan):
   candidate, fingerprints, vin, car_fw, source, exact_match = fingerprint(logcan, sendcan)
 
+  candidate = "CHEVROLET VOLT PREMIER 2017"
   if candidate is None:
     cloudlog.warning("car doesn't match any fingerprints: %r", fingerprints)
     candidate = "mock"
