diff --git a/panda/board/safety/safety_gm.h b/panda/board/safety/safety_gm.h
index b703f4721..847349078 100644
--- a/panda/board/safety/safety_gm.h
+++ b/panda/board/safety/safety_gm.h
@@ -8,6 +8,8 @@
 //      brake rising edge
 //      brake > 0mph
 
+bool pt_ecu_interceptor = false;
+
 const int GM_MAX_STEER = 300;
 const int GM_MAX_RT_DELTA = 128;          // max delta torque allowed for real time checks
 const uint32_t GM_RT_INTERVAL = 250000;    // 250ms between real time checks
@@ -42,7 +44,7 @@ const GmLongLimits *gm_long_limits;
 const int GM_STANDSTILL_THRSLD = 10;  // 0.311kph
 
 const CanMsg GM_ASCM_TX_MSGS[] = {{384, 0, 4}, {1033, 0, 7}, {1034, 0, 7}, {715, 0, 8}, {880, 0, 6},  // pt bus
-                                  {161, 1, 7}, {774, 1, 8}, {776, 1, 7}, {784, 1, 2},   // obs bus
+                                  {161, 1, 7}, {774, 1, 8}, {776, 1, 7}, {784, 1, 2}, {384, 1, 4}, {715, 1, 8}, {880, 1, 6}, {789, 1, 5}, {0x104c006c, 1, 3},  // obs bus
                                   {789, 2, 5},  // ch bus
                                   {0x104c006c, 3, 3}, {0x10400060, 3, 5}};  // gmlan
 
@@ -84,6 +86,15 @@ static int gm_rx_hook(CANPacket_t *to_push) {
 
   bool valid = addr_safety_check(to_push, &gm_rx_checks, NULL, NULL, NULL);
 
+  if (valid && (GET_BUS(to_push) == 1U)) {
+    int addr = GET_ADDR(to_push);
+    // Check for ASCM ECU Interceptor Status
+    // TODO: Add second check for chas bus interceptor. If only PT bus has an interceptor, we should only allow steering commands
+    if (addr == 885) {
+      pt_ecu_interceptor = true;
+    }
+  }
+
   if (valid && (GET_BUS(to_push) == 0U)) {
     int addr = GET_ADDR(to_push);
 
@@ -150,7 +161,7 @@ static int gm_rx_hook(CANPacket_t *to_push) {
     if (!gm_pcm_cruise && (addr == 715)) {
       stock_ecu_detected = true;
     }
-    generic_rx_checks(stock_ecu_detected);
+    generic_rx_checks(stock_ecu_detected && !pt_ecu_interceptor);
   }
   return valid;
 }
